ibis/expr/api.py

```diff
@@ -3425,6 +3426,7 @@ def _date_sub(left, right):
     day_of_week=_day_of_week,
     day_of_year=_extract_field('day_of_year', ops.ExtractDayOfYear),
     quarter=_extract_field('quarter', ops.ExtractQuarter),
+    epoch=_extract_field('epoch', ops.ExtractEpoch),
```

Comment:
epoch_seconds or seconds_since_epoch would be a more descriptive name i think.

Reply:
thanks @jreback , changed to `epoch_seconds`